multiqc/modules/mosdepth/mosdepth.py

```diff
@@ -15,6 +15,49 @@
 log = logging.getLogger(__name__)
 
 
+def read_config():
+    cfg = getattr(config, "mosdepth_config", dict())
+    if type(cfg) != dict:
+        return {}
+
+    cfg["include_contigs"] = cfg.get("include_contigs", [])
+    if type(cfg["include_contigs"]) != list:
+        cfg["include_contigs"] = []
+
+    cfg["exclude_contigs"] = cfg.get("exclude_contigs", [])
+    if type(cfg["exclude_contigs"]) != list:
+        cfg["exclude_contigs"] = []
+
+    cfg["xchr"] = cfg.get("xchr", None)
+    if type(cfg["xchr"]) != str:
+        cfg["xchr"] = None
+
+    cfg["ychr"] = cfg.get("ychr", None)
+    if type(cfg["ychr"]) != str:
+        cfg["ychr"] = None
+
+    if cfg["include_contigs"]:
+        log.info("Trying to include these contigs in mosdepth: {}".format(", ".join(cfg["include_contigs"])))
+    if cfg["exclude_contigs"]:
+        log.info("Excluding these contigs from mosdepth: {}".format(", ".join(cfg["exclude_contigs"])))
+    if cfg["xchr"]:
+        log.info('Using "{}" as X chromosome name'.format(cfg["xchr"]))
+    if cfg["ychr"]:
+        log.info('Using "{}" as Y chromosome name'.format(cfg["ychr"]))
```

Comment:
I think that `log.debug` is probably sufficient here. Or nothing - the log already contains some stuff about loaded config file changes, which I think should give a rough log of this already.

Reply:
True, I'll switch to `log.debug`

Here just tried to replicate the `samtools idxstats` behavior: https://github.com/ewels/MultiQC/blob/master/multiqc/modules/samtools/idxstats.py#L47-L66 But we can refactor that separately.